cmd/admin.go

```diff
@@ -59,7 +59,7 @@ func createLeaderboard(args []string, logger *zap.Logger) {
 	var metadata string
 
 	flags := flag.NewFlagSet("admin", flag.ExitOnError)
-	flags.StringVar(&dsns, "db", "root@localhost:26257", "CockroachDB JDBC connection details.")
+	flags.StringVar(&dsns, "database.address", "root@localhost:26257", "Address of CockroachDB server (username:password@address:port/dbname)")
```

Comment:
If we're renaming the input, let's also change the `dsns` variable name to stay consistent.